### DISCUSSION TITLE
Aggregate Values in ClassMap

### DISCUSSION THREAD

**Discussion by binazhar** [JoshClose/CsvHelper#1498] [Q&A]
I am trying to read multiple records types from a csv file where the first line have following records

id, name, Number of values

and the second line have

date, shift, values[0..n], quality

With the help from original csvhelper example i can read the original lines but i can not figure out how to pass BarMap the number of values that need to be aggregated and then how to sum them up. 

_fooRecords.Add(csv.GetRecord<Foo>());_ Foo.numValue <- Number of values

_barRecords.Add(csv.GetRecord<Bar>());_

public sealed class BarMap : ClassMap<Bar>
{
    public BarMap()
    {
        Map(m => m.date).Index(1);
        Map(m => m.shift).Index(2);
        //Aggregation of values 1 ..n goes here
        Map(m => m.quality).Index(n+1);

    }
}

I would appreciate if anyone can help me in this.

**Comment by JoshClose** [comment]
I answered your question on StackOverflow. https://stackoverflow.com/questions/61107278/csvhelper-c-list/61123883#61123883
